rpcserver.go

```diff
@@ -2119,6 +2119,11 @@ func createRPCInvoice(invoice *channeldb.Invoice) (*lnrpc.Invoice, error) {
 		fallbackAddr = decoded.FallbackAddr.String()
 	}
 
+	settleDate := int64(0)
```

Comment:
The Unix timestamp for the zero value of `time.Time` is -62135596800, which I think is not as friendly as a default of 0.